pkg/utils/has/controller.go

```diff
@@ -495,3 +495,47 @@ func (h *SuiteController) GetHasComponentConditionStatusMessages(name, namespace
 	}
 	return
 }
+
+// CreateSnapshotEnvironmentBinding creates a new SnapshotEnvironmentBinding
+func (h *SuiteController) CreateSnapshotEnvironmentBinding(name, namespace, applicationName, snapshotName, environmentName, componentName string) (*appservice.SnapshotEnvironmentBinding, error) {
+	bindingComponents := make([]appservice.BindingComponent, 0)
+	snapshotEnvironmentBinding := &appservice.SnapshotEnvironmentBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Spec: appservice.SnapshotEnvironmentBindingSpec{
+			Application: applicationName,
+			Environment: environmentName,
+			Components: append(bindingComponents,
+				appservice.BindingComponent{
+					Configuration: appservice.BindingComponentConfiguration{
+						Replicas: 1,
```

Comment:
In the future we need to customize the replicas... Can you change it to make it more 'customizable'?

Reply:
Yes, of course. Thank you!